### ISSUE TITLE
Sed in-place edit incompatibility on OSX

### ISSUE THREAD

**Issue by jthurst3** [clehner/mksite#1]
I have a simple `_nav.txt` file that contains the following contents:

```
. About
apply.html Apply
contact.html Contact
faq.html FAQ
```

After making a small modification to `faq.md` and running `mksite`, I get the following error: 

```
rendering nav
sed: 1: "_nav.html": invalid command code _
make: *** [README.nav.html] Error 1
rm README.nav.html README.content.html
```

I believe this error is similar to what's described [here](http://stackoverflow.com/questions/7573368/in-place-edits-with-sed-on-os-x), as I'm running on OSX and using BSD sed from May 10, 2005. Any ideas on how to fix?

**Comment by clehner** [comment]
It's probably because of `\s` and `\S`, which the 2005 sed doesn't support. bb also had this issue. I just pushed a sed-fix branch that uses `[ \t]` instead. Does that work?

**Comment by jthurst3** [comment]
Doesn't seem to work. I get the same output.

**Comment by clehner** [comment]
> sed: 1: "_nav.html": invalid command code _

looks like it's interpretting _nav.html as a command instead of as a filename. I'm replacing the filename now with <filename to put it in as stdin instead of as an argument. I'm testing with plan9 sed on Debian and see some other compatibility issues too.

**Comment by clehner** [comment]
How about now?

**Comment by jthurst3** [comment]
Compiles and generates the static pages, but it is not correctly displaying bold text. Here is some sample input and output:
`sample.md`:

```
**This should be in bold.**
```

relevant part of `sample.html`:

```
<p><strong></strong>This should be in bold.<strong></strong></p>
```

**Comment by clehner** [comment]
Ok, this would then be an issue with `md2html.awk`. I'm not sure what to do about that, other than suggest using a different markdown-to-html tool.

Edit: squashed and rebased the sed-fix branch onto master.

**Comment by clehner** [comment]
Yeah, md2html.awk seems to be relying on some GNU awk feature. I see the same result as you when using plan9's awk. Do you have gawk?
